CedarJavaFFI/src/interface.rs

```diff
@@ -623,3 +623,33 @@ fn policies_str_to_pretty_internal<'a>(
         }
     }
 }
+
+#[cfg(test)]
+mod interface_tests {
+    use super::*;
+    use crate::jvm_test_utils::*;
+    use jni::JavaVM;
+    use std::sync::LazyLock;
+
+    // Static JVM to be used by all the tests. LazyLock for lazy thread-safe lazy initialization
+    static JVM: LazyLock<JavaVM> = LazyLock::new(|| create_jvm().unwrap());
+
+    mod policy_tests {
+        use super::*;
+
+        fn policy_effect_test_util(env: &mut JNIEnv, policy: &str, expected_effect: &str) {
```

Comment:
```suggestion
        #[track_caller]
        fn policy_effect_test_util(env: &mut JNIEnv, policy: &str, expected_effect: &str) {
```

Always nice to add for this sort of utility

https://doc.rust-lang.org/reference/attributes/codegen.html#the-track_caller-attribute